src/SickSafetyscannersRos.cpp

```diff
@@ -382,7 +382,12 @@ SickSafetyscannersRos::createLaserScanMessage(const sick::datastructure::Data& d
 {
   sensor_msgs::LaserScan scan;
   scan.header.frame_id = m_frame_id;
-  scan.header.stamp    = ros::Time::now();
+
+  std::shared_ptr<sick::datastructure::DataHeader> data_header = data.getDataHeaderPtr();
```

Comment:
I guess this can be const (or const to const) as we are just using getters, but it is a minor thing.